src/main/java/com/michalsydoryk/app/board/Board.java

```diff
@@ -0,0 +1,17 @@
+package com.michalsydoryk.app.board;
+
+import com.michalsydoryk.app.sign.Sign;
+
+import java.util.Set;
+
+public interface Board<T> {
+    int MAX_SIZE = Integer.MAX_VALUE;
```

Comment:
Another public, it's nice that you have interface for the package but lets see whats inside this package..